ambari-project/pom.xml

```diff
@@ -35,7 +35,7 @@
     <ldap-api.version>1.0.0</ldap-api.version>
     <checkstyle.version>8.9</checkstyle.version>
     <swagger.version>1.5.19</swagger.version>
-    <swagger.maven.plugin.version>3.1.4</swagger.maven.plugin.version>
+    <swagger.maven.plugin.version>3.1.5</swagger.maven.plugin.version>
```

Comment:
Any reason for not going with latest 3.1.7?

Reply:
I wanted a version that fixes my issue